characters/Character.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
+
+const charSchema = new mongoose.Schema({
+    charName: {
+        type: String,
+        required: true,
+        unique: true
+    },
+    birthday: {
+        required: true,
+        type: String
+    },
+    wand:{
+        type: String,
+    },
+    house: {
+        required: true,
+        type: String,
+        validate: {
+            validator: /(Gryffindor| Slytherin| Hufflepuff| Ravenclaw)/, 
+            msg: "invalid house"
+        }
+    }
+});
+
+// charSchema.pre('save', function (){
```

Comment:
didn't have a need for it because I wasn't hashing anything. Decided not to both with username and passwords but instead do character profiles. Kept it though because it would be good reference for later.